test/test_models.py

```diff
@@ -328,6 +329,18 @@ def _check_input_backprop(model, inputs):
     _model_params[m] = {"input_shape": (1, 3, 64, 64)}
 
 
+# skip big models to reduce memory usage on CI test
+skipped_big_models = {
+    "vit_h_14",
+    "regnet_y_128gf",
+}
+
+
+def do_skip_big_model(model_name):
```

Comment:
Nit: Since we don't use this more than once, we can lose the extra method and move the check directly on the test.